plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtStoragePoolDef.java

```diff
@@ -55,6 +59,7 @@ public String toString() {
     private String _authUsername;
     private AuthenticationType _authType;
     private String _secretUuid;
+    private Map<String, String> _nfsopts = new HashMap<>();
```

Comment:
```suggestion
    private Map<String, String> _nfsMountOpts = new HashMap<>();
```

update wherever applicable

Reply:
Done